public/stylesheets/style.css

```diff
@@ -304,3 +324,55 @@ input[type="submit"]:hover {
   bottom: 2.5%;
   z-index: 2;
 }
+
+/* Styling changes */
+h1 {
```

Comment:
we want all these changes for all h1..h5 tags?

Reply:
Yes, no other changes and its only the font-weight and slightly off black

Reply:
Also h1, 2, 4, 5 are also only small changes. h4 + 6 are still available. I could refactor to change only the specific parts of text but then everything is inconsistent. I will refactor padding on h5 out